server.js

```diff
@@ -442,9 +462,14 @@ if (config.startServer) {
         function(callback) {
             sprocs.init(function(err) {
                 if (ERR(err, callback)) return;
-                callback(null);
-            });
-        },
+				if ('migrate-and-exit' in argv && argv['migrate-and-exit']) {
```

Comment:
Can we shift this to another separate stanza in the big `async.series()` list? Also, this seems to be using tabs rather than spaces for indentation. I wonder why the linter didn't complain?